packages/debugger/lib/controller/sagas/index.js

```diff
@@ -211,6 +212,22 @@ function* stepOver() {
   );
 }
 
+/**
+ * runToEnd - run the debugger till the end
+ */
+function* runToEnd() {
+  var finished;
+
+  do {
+    yield* advance();
+
+    finished = yield select(controller.current.trace.finished);
+    if (finished) {
+      break; //can break immediately if finished
+    }
```

Comment:
This is redundant, since we check the `while(!finished)` immediately following this.
```suggestion
```

Reply:
Oh yes, I will remove that part and make the changes.